pkg/ssh/server/ssh.go

```diff
@@ -50,13 +51,28 @@ func NewServer(addr string, hostKey []byte, keys []ssh.PublicKey, log log.Logger
 				log.Debugf("attempt to bind %s:%d - %s", host, port, "granted")
 				return true
 			},
+			ReverseUnixForwardingCallback: func(ctx ssh.Context, socketPath string) bool {
+				log.Debugf("attempt to bind socket %s", socketPath)
+
+				_, err := os.Stat(socketPath)
+				if err == nil {
+					log.Debugf("%s already exists, removing", socketPath)
+
+					_ = os.Remove(socketPath)
+				}
```

Comment:
why do we delete it? And where the binding happens?

Reply:
That's the behaviour of OpenSSH with the `StreamLocalBindUnlink=yes`  option
Which is useful in our case to ensure we don't have dangling (old) sockets